### PR TITLE
Add tutorial i18n guidelines to `TRANSLATING.md`

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- New or updated content

#### Description

Added a new section to "Translation Tips" with guidelines for tutorial translations.

### CODE REVIEW

File: TRANSLATING.md [hunk 1]
```diff
@@ -308,23 +308,32 @@ However, these pages are translated directly here and **these warnings are not m
...
 
+### Tutorials
+
+Some of our pages are tutorials, which are a linear sequence of lessons made to introduce users to Astro concepts. They also include custom components that you can find the labels for translation in the language's `i18n/ui.ts` file, as described in the [Translation Structure section](#2-ui-text).
+
+Considering it's nature, we cannot merge partial tutorial translations, only fully translated ones. We care about making our contribution workflow sustainable, so we've made a few guidelines to avoid burdening any contributor working on them:
+
+- Tutorial translation PRs are recommended to contain 1-3 lessons (each lesson is its own Markdown file) or a full unit if small (equivalent to a chapter of the tutorial), this allows reviewers to have an easier time as opposed to having them to review the entire tutorial in one go.
+
+- All translations are going to be merged in a custom branch and then merged into `main` once the tutorial is 100% complete. No need to change anything in your PRs, keep them targetting `main` and we will take care of creating and changing the merge target to the custom branch set up by us.
```
Comment (sarah11918) [inline TRANSLATING.md:319]:
```suggestion
Some of our pages belong to tutorials, which are a linear sequence of lessons to introduce users to Astro concepts. They also include custom components that you can find the labels for translation in the language's `i18n/ui.ts` file, as described in the [Translation Structure section](#2-ui-text).

Considering how distracting it would be to try to follow a tutorial with only some translated pages, we cannot merge partial tutorial translations, only fully translated ones. But, that's a huge job for one translator! We care about making our translator workflow sustainable, so we've made a few guidelines specifically for the tutorial pages to avoid burdening any contributor working on them:

- Tutorial translation PRs are recommended to contain 1-3 lessons (each lesson is its own `.mdx` file) or a full unit if small (equivalent to a chapter of the tutorial). This is not only to help translators work in smaller units, but also allows reviewers to have smaller PRs to review.

- All translations will be merged in a custom branch first, and then merged into `main` once the tutorial is 100% complete. No need to change anything in your PR workflow! Your PRs should be to the `main` branch, like any other translation PR, and we will take care of creating and changing the merge target to the custom branch set up by us. This allows the tutorial to be translated in smaller pieces, in the same way you would translate any other page, without making these changes go live on the docs site until the tutorial is fully translated.
```

**Review by sarah11918 — APPROVED** [review]
Thank you so much, Yan!  :partying_face: This is going to be so helpful to have now that translation bot is pushing the tutorial pages in everyone's faces!  :laughing:

Made some suggestions for you to look at!

File: TRANSLATING.md [hunk 2]
```diff
@@ -195,10 +196,9 @@ Translators are free to create and mantain a glossary, style guide and other tip
 
 Feel free to take a look at the [Deutsch Guide](https://github.com/withastro/docs/blob/main/src/i18n/de/README.md) for an example.
 
-
 ### Frontmatter
 
-Our pages are generated from Markdown files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data. 
+Our pages are generated from Markdown files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data.
```
[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply TRANSLATING.md:201]: I think a peep looking to translate our docs will already know what MDX is, so I don't think we need to explain it, I'd just change to MDX, IMO. Feel free to convince me otherwise!
```suggestion
Our pages are generated from MDX files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data.
```

File: TRANSLATING.md [hunk 3]
```diff
@@ -212,13 +212,14 @@ i18nReady: true
...
 ```
+
 **tl/dr: Translate only some values, never translate properties!**
 
 The frontmatter **properties** themselves, like `title` and `description`, should not be translated, as doing so would cause a runtime error and break our CI.
 
 The only frontmatter **values** that should be translated are those corresponding to the `title` and `description` properties: "*Data Fetching*" and "*Learn how to fetch remote data with Astro using the fetch API*."
 
-Other frontmatter properties that aren't mentioned here should be ignored and not translated, as we use them for handling our "Edit this page" links or for specifying in which category an integration belongs, etc. 
+Other frontmatter properties that aren't mentioned here should be ignored and not translated, as we use them for handling our "Edit this page" links or for specifying in which category an integration belongs, etc.
```
[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply TRANSLATING.md:222]: I will add this to the "Components" heading!

[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply TRANSLATING.md:222]: Just saw we actually mention it in there, just updated to not talk about being in the frontmatter

**Review by dreyfus92 — APPROVED** [review]
LGTM ! ✅
